parsl/executors/high_throughput/executor.py

```diff
@@ -630,7 +630,8 @@ def submit(self, func, resource_specification, *args, **kwargs):
 
         The outgoing_q is an external process listens on this
         queue for new work. This method behaves like a
-        submit call as described here `Python docs: <https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.ThreadPoolExecutor>`_
+        submit call as described here:
+         `Python docs: <https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.ThreadPoolExecutor>`_
```

Comment:
when the CI system goes to test building documentation from this, it fails with this error:

```
/home/runner/work/parsl/parsl/parsl/executors/high_throughput/executor.py:docstring of parsl.executors.high_throughput.executor.HighThroughputExecutor.submit:6:Unexpected indentation.
```

You can see that error yourself by clicking on the red x in the checks section of your pull request, and then scrolling through to find the error near the end.

Probably that comes from this line 634 being indented by one character more than the rest of the paragraph before it.

Reply:
Okay , I can see this let me review it